manifests/a/Albion/Online/24.8.21/Albion.Online.locale.en-US.yaml

```diff
@@ -0,0 +1,21 @@
+# Created with komac v2.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Albion.Online
+PackageVersion: 24.8.21
+PackageLocale: en-US
+Publisher: Sandbox Interactive GmbH
+PublisherUrl: https://albiononline.com/
+PrivacyUrl: https://albiononline.com/privacy_policy
+PackageName: Albion Online
+PackageUrl: https://albiononline.com/download
+License: Proprietary
+LicenseUrl: https://albiononline.com/en/terms_and_conditions
```

Comment:
> Url validation error
> 
> * manifests\a\Albion\Online\24.8.21
>   * https://albiononline.com/
>     * Http status code: Forbidden
>   * https://albiononline.com/privacy_policy
>     * Http status code: Forbidden
>   * https://albiononline.com/download
>     * Http status code: Forbidden
>   * https://albiononline.com/en/terms_and_conditions
>     * Http status code: Forbidden
```suggestion
# PublisherUrl: 
# PrivacyUrl: 
PackageName: Albion Online
# PackageUrl: 
License: Proprietary
# LicenseUrl:
```

Reply:
Or just waive these errors. The urls are all correct and will work fine when an user clicks them.

Reply:
I'd prefer keeping them in the manifest, but add comment them out to prevent validation errors.

```suggestion
# PublisherUrl: https://albiononline.com/
# PrivacyUrl: https://albiononline.com/privacy_policy
PackageName: Albion Online
# PackageUrl: https://albiononline.com/download
License: Proprietary
# LicenseUrl: https://albiononline.com/en/terms_and_conditions
```